### ISSUE TITLE
Hi , Is this historical data already updating?

### ISSUE THREAD

**Issue by robyle** [dmikushin/binance-historical-data#1]
How can I get the new binance histroical data and recently data ？

**Comment by dmikushin** [comment]
Hi @robyle the dataset offered here is stale, but it's highly probable that you could still scrape the new history with the existing bihistorian script. The advantage of bihistorian over some Python scripts available on the Internet is C++ native code and multithreaded parallel dumping of historical data for multiple pairs in parallel.

**Comment by robyle** [comment]
> Hi @robyle the dataset offered here is stale, but it's highly probable that you could still scrape the new history with the existing bihistorian script. The advantage of bihistorian over some Python scripts available on the Internet is C++ native code and multithreaded parallel dumping of historical data for multiple pairs in parallel.

yeah, I have already writ a logic using C# and insert to mongodb. I just want find there weather exist a stability project ,which can run into database.
